stable/rabbitmq-ha/README.md

```diff
@@ -138,6 +138,10 @@ and their default values.
 | `rbac.create`                                  | If true, create & use RBAC resources                                                                                                                                                                  | `true`                                                     |
 | `replicaCount`                                 | Number of replica                                                                                                                                                                                     | `3`                                                        |
 | `resources`                                    | CPU/Memory resource requests/limits                                                                                                                                                                   | `{}`                                                       |
+| `securityContext.enabled`                      | Enable security context                                                                                                                                                                               | `false`                                                    |
```

Comment:
Let's remove enabled and default to the new more secure permissions. Rabbitmq already drops permission at runtime to 100 via setuid anyway.

Reply:
Done